### PR TITLE
add parse env to init2

### PR DESCRIPTION
**Pull request by lin755**

### CODE REVIEW

**Comment by gaozhangfei** [comment]
+1

**Comment by gaozhangfei** [comment]
+1

**Review by lin755 — COMMENTED** [review]
#551

File: wd_util.c [hunk 1]
```diff
@@ -2233,7 +2233,7 @@ void *wd_dlopen_drv(const char *cust_lib_dir)
 		if (ret < 0)
 			goto free_node;
 
-		node->dlhandle = dlopen(lib_path, RTLD_NOW);
+		node->dlhandle = dlopen(lib_path, RTLD_NODELETE | RTLD_NOW);
```
Comment (Liulongfang) [inline wd_util.c:2234]: After modifying the problem of dlopen, please modify the supporting dlclose together:
Line 1998: dlclose(dlnode->dlhandle);

Comment (lin755) [reply wd_util.c:2234]: [dae358e](https://github.com/Linaro/uadk/pull/552/commits/dae358ecb5ed5ca82fe138e86081a79dfee1c111)

**Review by Liulongfang — APPROVED** [review]
OK

**Pull request merged**